spec/controllers/application_drafts_controller_spec.rb

```diff
@@ -168,7 +168,7 @@
 
       context 'for a valid draft' do
         before do
-          allow_any_instance_of(ApplicationDraft).to receive(:valid?).with(:apply).and_return(true)
+          allow_any_instance_of(ApplicationDraft).to receive(:ready?).and_return(true)
```

Comment:
This is true! However, the spec indeed fails as we're missing the ability to create an `Application` from an `ApplicationDraft`. I'll investigate.